frontend/src/_layouts/base.html

```diff
@@ -79,45 +79,24 @@
       </div>
       <!-- Menu items -->
       <div data-menu-toggled class="hidden sm:block">
-        <div
-          class="child-a:px-2 child-a:py-1 font-header flex flex-col items-end justify-end space-y-4 py-1 pl-36 text-sm font-semibold text-gray-600 sm:flex-row sm:items-stretch sm:justify-start sm:space-x-1 sm:space-y-0 md:pl-56 lg:justify-center lg:space-x-3 lg:pl-72 lg:pr-28 lg:text-base"
+        <div class="child-a:px-2 child-a:py-1 font-header flex flex-col items-end justify-end space-y-4 py-1 pl-36 text-sm font-semibold text-gray-600 sm:flex-row sm:items-stretch sm:justify-start sm:space-x-1 sm:space-y-0 md:pl-56 lg:justify-center lg:space-x-3 lg:pl-72 lg:pr-28 lg:text-base"
         >
-          <!--
-            Currently Parcel keeps html extension in the references.
-            https://github.com/parcel-bundler/parcel/discussions/6903
-            Possible solution: https://v2.parceljs.org/languages/html/#:~:text=This%20example%20uses%20posthtml-shorten%20to%20shorten%20URLs%20using%20a%20custom%20shortening%20function.
-          -->
-          <!-- Currently it is not possible to pass data from page to the layout: https://github.com/posthtml/posthtml-extend/issues/21 -->
-          <!-- Therefore override corresponding *-menu-item block in a page to make it styled active -->
-          <!-- In the future we can leverage PUG: https://stackoverflow.com/a/29089926/3082178 -->
-          <block name="audios-menu-item">
-            <a href="/">Audios</a>
-          </block>
-          <block name="memories-menu-item">
-            <a href="/memories">Memories</a>
-          </block>
-          <a href="https://www.audioseva.com/donate" target="_blank">Donate</a>
-          <a href="https://docs.google.com/forms/d/e/1FAIpQLSfjwP_n5uJPD-BxwE_Ta-tbaf2aWNWZqe-YOOoUzjcPw209Uw/viewform" target="_blank">Contact</a>
-          <block name="about-menu-item">
-            <a href="/about">About</a>
-          </block>
-          <block name="join-menu-item">
-            <a href="/join">Join</a>
-          </block>
-        </div>
+          {% assign navPages = collections.all | eleventyNavigation %}
+          {% for entry in navPages %}
+            <a href="{{ entry.url }}" {% if entry.url == page.url %} class="rounded-md bg-teal-800 text-white"{% endif %}  {% if entry.title == "donate" %} target="_blank" {% endif %}  {% if entry.title == "contact" %} target="_blank" {% endif %} > {{ entry.title }} </a>
```

Comment:
Here we need a more generic way of specifying `target _blank`. The layout should not know about the pages, otherwise there's a violation of the Separate of Concerns principle. Try adding some flag to the frontmatter. I beleive you will be able to consume that data in the collection entry: https://www.11ty.dev/docs/collections/#collection-item-data-structure.

Reply:
this is changed by passing a flag `target` to the frontmatter, in the collection entry.